configs/machines/generic.yaml

```diff
@@ -1,44 +1,158 @@
+#######################################################################################
+# The dummy yaml machine config file for esm-tools
+#
+# This is a dummy machine file. It is intend to be used for creating a new config file 
+# for a new machine setup. It provides a minimal structure to be filled in with
+# the oppropriate values of the new HPC system. 
+#
+# Usage:  
+#     - Please replace the placeholder indicated by <...> 
+#       with your PHC specific values
+#     - Please uncomment sections of this yaml file if appropriate.
+#       Possible sections are indicating this (see below).
+#     - See also files for other machines that are already implemented in esm-tools.
+#     - After editing this file, save it as <machine>.yaml. Add also an entry in
+#       the file all_machines.yaml (in this folder) for the new machine.
+#
+# Last changes: N. Wieters, May 2023: Added comments and placeholders.
+#######################################################################################
+
 # GENERIC YAML CONFIGURATION FILES
+#
+# Set hostname of the machine 
+name: <hostname>
 
-name: generic
-accounting: false
-hetjob_flag: "?" # either hetjob or packjob
-batch_system: "slurm"
+# General information about operating system
+# Operating system
+#operating_system: "<your-operating-system>"
+# Shell interpreter (e.g. "/usr/bin/bash")
+sh_interpreter: "<path-to-shell>"
 
-jobtype: compute
+# Information about the job scheduler
+# Set batch system (e.g. slurm)
+batch_system: "<your-job-scheduler>"
 
-partition: generic 
+# Set whether an account needs to be set for batch system (e.g. true or false). The actual account that will be used for the experiment will be set in the runscript.
+accounting: true 
+#
+#hetjob_flag: "?" # either hetjob or packjob
+#
+#jobtype: compute
+#
+# Information about the partitions of the machine
+#
+# Specify if hyperthreading should be used.
+use_hyperthreading: False
+# Set further options depending on hyperthreading variable 'use_hyperthreading' by uncommenting and editing the following 'choose_use_hyperthreading' section:
+#choose_use_hyperthreading:
+#        True: 
+#            hyperthreading_flag: ""
+#            mt_launcher_flag: ""
+#        False: 
+#            <set_any_variable_here_related_to_hypertherading>: <value>
+#        "anyother_scalar_value':
+#            <set_any_variable_here_related_to_hypertherading>: <value>
```

Comment:
In this instance, I'd leave the "1" and "0" as in Levante. That was only to prevent that someone defined`use_hyperthreading` as a 0 string and it will still be evaluated as true.

Reply:
Thanks Miguel, I will remove the option for a scalar value.